ddtrace/llmobs/_integrations/langgraph.py

```diff
@@ -132,6 +136,23 @@ def _link_task_to_parent(self, task_id, task, finished_task_names_to_ids):
             span_links.append(span_link)
 
 
+def _normalize_triggers(triggers, finished_tasks, next_task) -> List[str]:
+    """
+    Return the default triggers for a LangGraph node.
+
+    For nodes queued up with `langgraph.types.Send`, the triggers are an unhelpful ['__pregel_push'].
+    In this case (and in any case with 1 finished task and 1 trigger), we can infer the trigger from
+    the one finished task.
+    """
+    if len(finished_tasks) != 1 or len(triggers) != 1:
+        return triggers
+
+    finished_task_name = list(finished_tasks.keys())[0]
```

Comment:
should just check that len > 0 here

Reply:
actually we should be safe here since `0 != 1` would cause us to return early above. i think at this point both `len(finished_tasks)` and `len(triggers)` are `1`

Reply:
ah yes, good point